### ISSUE TITLE
Adding `-D` when also passing config file ignores config?

### ISSUE THREAD

**Issue by srenatus** [golangci/golangci-lint#66]
With a `.golangci.toml` containing
```toml
[linters]
enable-all = true
disable = [
  "dupl",
  "goconst",
  "maligned",
  "golint"
]
```
and running `golangci-lint run -c ../../.golangci.toml -Dtypecheck ./...` to *also disable `typecheck`*, the previously disabled linters are enabled again.

**Comment by jirfag** [comment]
hi, thank you for the issue!
I have a fix in the linked commit, but it needs more testing

**Comment by srenatus** [comment]
✔️ Thanks! Works fine.

**Comment by jirfag** [comment]
I included this fix into [v1.5 release](https://github.com/golangci/golangci-lint/releases/tag/v1.5)
